strings.go

```diff
@@ -306,3 +315,51 @@ func keyStmt(key []byte) *ast.GenDecl {
 		}},
 	}
 }
+
+func strConstBlacklist(node ast.Node, info *types.Info, blacklist map[types.Object]struct{}) {
```

Comment:
s/strConst/const/, since the feature is generally about literals, not just strings

Reply:
ok